rfd/0035-desktop-access-windows-authn.md

```diff
@@ -0,0 +1,119 @@
+---
+authors: Andrew Lytvynov (andrew@goteleport.com)
+state: draft
+---
+
+# RFD 35 - Desktop Access - Windows Certificate Authentication
```

Comment:
I am far from being an Active Directory expert, but wouldn't this require the installation of Teleport CA in the Active Directory's NTAuth store? Have you considered the implications of that from a UX standpoint? I don't think there're better solutions, but I want to make sure that this was on your radar.

Reply:
Correct. The only way to teach AD to trust smartcard certs is to import the CA cert into the trust store.
I have not looked at how exactly that process works yet, and it will probably be kludgy. Likely steps 1-4 of https://docs.microsoft.com/en-us/troubleshoot/windows-server/windows-security/enabling-smart-card-logon-third-party-certification-authorities#configuration-instructions

But it's the only way I found to not rely on usernames/passwords for RDP logins.